grpc/server_test.go

```diff
@@ -0,0 +1,293 @@
+package sentrygrpc_test
+
+import (
+	"context"
+	"errors"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+	sentrygrpc "github.com/getsentry/sentry-go/grpc"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	grpchealth "google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
Is it relevant for the tests that we use this package, or can we use `google.golang.org/grpc` by itself?

Reply:
Can you elaborate?

As far as I understand, to test grpc server we need a service to serve (in this case, I used grpc_health_v1 service). The service doesn't matter but grpc_health_v1 looked the easiest because it is bundled with grpc.

Reply:
I just looked again at pkg.go.dev, for some reason I thought this was in a separate module, but it is not.
It's okay to use it.